tests/integration_tests/cmd/test_analyze.py

```diff
@@ -0,0 +1,36 @@
+# This file is part of cloud-init. See LICENSE file for license information.
+"""Tests for `cloud-init analyze`"""
+import pytest
+
+from cloudinit.distros import uses_systemd
+from tests.integration_tests.integration_settings import PLATFORM
+from tests.integration_tests.util import get_datetime_from_string
+
+
+class TestAnalyzeCommand:
+    @pytest.mark.skipif(not uses_systemd(), reason="Relies on systemd output")
```

Comment:
Not sure if this is worth using a production function `uses_systemd` or whether I should just use `tests.integration_tests.releases.IS_UBUNTU`

Reply:
as-is seems fine.